packages/specs/schema/src/domain/JsonMedia.ts

```
@@ -1,23 +1,45 @@
+import {isArray, isClass} from "@tsed/core";
 import {OS3MediaType} from "@tsed/openspec";
 
+import {Partial} from "../decorators/operations/partial.js";
 import {JsonMap} from "./JsonMap.js";
 import {JsonSchema} from "./JsonSchema.js";
 
 export class JsonMedia extends JsonMap<OS3MediaType<JsonSchema>> {
   $kind: string = "operationMedia";
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)